### DISCUSSION TITLE
CreateCustomToken like firebase

### DISCUSSION THREAD

**Discussion by sanmit-vartak** [supabase/supabase#20025] [Feature Requests]
I was looking out for solution to fit in my current firebase use-case into supabse.
Use case: 
1. Create Custom token for user using UID/email from backend service account
2. Sign in user with custom toke using provided same

PS: I do not want user to sign in by himself. since I am allowing anonymous users to use the service without login, and also for my logged users CRM database is separate so keeping their passwords in supabase as well will be redundant

**Comment by sanmit-vartak** [comment]
@hf

**Comment by jpiabrantes** [comment]
Would also love this. Right now I am doing my Auth in Firebase just because of this feature.
